<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QDialog" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>320</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="lbN1">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Первый операнд: </string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbN2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>50</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Второй операнд: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbRezt">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>90</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Результат: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edN1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>131</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edN2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>131</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edRezt">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>131</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btSum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сложение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btSub">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вычитание</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btDiv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Деление</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btMul">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Умножение</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
